lib/course_planner_web/controllers/auth/api_session_controller.ex

```diff
@@ -0,0 +1,41 @@
+defmodule CoursePlannerWeb.Auth.Api.V1.JsonSessionController do
```

Comment:
Not sure about the `V1` in the module, the nesting and the directory structure doesn't match. Can you fix that?

Reply:
What do you suggest cause i have no idea, i just saw this as a suggestion. 
1 - create the folder structure
2 - remove V1

Reply:
For now we can remove the V1 from the module. Internally we can rename/move anytime, we should only leave the route consistent.

Reply:
done